components/backup-stream/src/router.rs

```diff
@@ -1166,28 +1277,27 @@ struct DataFile {
     resolved_ts: TimeStamp,
     min_begin_ts: Option<TimeStamp>,
     sha256: Hasher,
-    inner: BufWriter<File>,
+    inner: ZstdEncoder<BufWriter<File>>,
```

Comment:
Please add some config for changing the compression method or at lease use lightweight compress algorithm (like `lz4`) instead of `zstd`. Or I'm afraid we may have heavy performance impact.

Reply:
Compare with the v6.2.0, their cpu usage are very closely.